beacon/beacon.go

```diff
@@ -1,9 +1,14 @@
 package beacon
 
 import (
+	"errors"
+
 	"github.com/alethio/eth2stats-client/types"
 )
 
+// Error to use when a  call is not available
+var NotAvailable = errors.New("Feature is not available")
```

Comment:
What do you think about `(Feature)NotImplemented` rather than `NotAvailable`? 
In my head, not available might also mean the node is not available or not able to handle the request while "not implemented" is pretty clear.

Reply:
makes sense, done